.github/workflows/codeql-analysis.yml

```diff
@@ -0,0 +1,72 @@
+# For most projects, this workflow file will not need changing; you simply need
+# to commit it to your repository.
+#
+# You may wish to alter this file to override the set of languages analyzed,
+# or to provide custom queries or build logic.
+#
+# ******** NOTE ********
+# We have attempted to detect the languages in your repository. Please check
+# the `language` matrix defined below to confirm you have the correct set of
+# supported CodeQL languages.
+#
+name: "CodeQL"
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    # The branches below must be a subset of the branches above
+    branches: [ master ]
+  schedule:
+    - cron: '16 18 * * 5'
```

Comment:
I can see how cron schedule could be useful for repos that don't change often. In our case, the repo is quite active and master pushes will trigger scanner. Up to you if you want to keep this Friday evening schedule.

Reply:
OK, lets remove it

Reply:
I think we should use cron instead of push to master as codeql takes quite long to finish ~90min

Reply:
Sure, no concerns from me.